### PR TITLE
Documentation: Kedro-Viz in VSCode Kedro extension

### PR DESCRIPTION
**Pull request by jitu5**
## Description

Resolves https://github.com/kedro-org/vscode-kedro/issues/89

## Development notes

- Updated README.md
- Updated kedro-viz_visualisation.md

## QA notes

- All tests should pass

## Checklist

- [X] Read the [contributing](/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added new entries to the `RELEASE.md` file
- [ ] Added tests to cover my changes

### CODE REVIEW

**Comment by astrojuanlu** [comment]
RTD build failing 

```
sphinx.errors.SphinxWarning: /home/docs/checkouts/readthedocs.org/user_builds/kedro-viz/checkouts/2078/docs/source/kedro-viz_visualisation.md:250:broken link: https://github.com/kedro-org/vscode-kedro?tab=readme-ov-file#how-to-use-this-extension (Anchor 'how-to-use-this-extension' not found)
```

**Review by astrojuanlu — APPROVED** [review]
Amazing 👏🏼

File: docs/source/kedro-viz_visualisation.md [hunk 1]
```diff
@@ -238,4 +238,23 @@ import '@quantumblack/kedro-viz/lib/styles/styles.min.css';
 const MyApp = () => <div style={{height: `100vh`}}><KedroViz data={json} options={options} /></div>;
 ``` 
 
-For more information on how to use kedro as React component with all possible props please refer to [Kedro-Viz on npm](https://www.npmjs.com/package/@quantumblack/kedro-viz)
\ No newline at end of file
+For more information on how to use kedro as React component with all possible props please refer to [Kedro-Viz on npm](https://www.npmjs.com/package/@quantumblack/kedro-viz)
```
Comment (rashidakanchwala) [inline docs/source/kedro-viz_visualisation.md:241]:
```suggestion
For more information on how to use kedro as React component with all possible props, see [Kedro-Viz on NPM](https://www.npmjs.com/package/@quantumblack/kedro-viz)
```

Comment (noklam) [reply docs/source/kedro-viz_visualisation.md]: Is it not `npm`? I thought I see this more common, but not sure.

**Review by noklam — APPROVED** [review]

**Pull request merged**